src/cryptography/hazmat/_oid.py

```diff
@@ -2,9 +2,11 @@
 # 2.0, and the BSD License. See the LICENSE file in the root of this repository
 # for complete details.
 
+import typing
+
 
 class ObjectIdentifier(object):
-    def __init__(self, dotted_string: str):
+    def __init__(self, dotted_string: str) -> None:
```

Comment:
The [mypy cheatcheet|https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html] consistently annotates the return type of constructors, so I'm doing here as well. If you don't want this, please just say so.